airflow/providers/qubole/example_dags/example_qubole_sensor.py

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
Unrelated?

Reply:
Yes - sorry I accidentally included this after a rebase. When I squash/rebase this should be resolved